### ISSUE TITLE
Map does not show up with latest version of Xamarin.GooglePlayServices

### ISSUE THREAD

**Issue by GuiF007** [amay077/Xamarin.Forms.GoogleMaps#639]
### VERSIONS

* Xamarin.Forms.GoogleMaps - 3.2.0
* Xamarin.Forms - 4.1.0.x
* Xamarin.GooglePlayServices - 71.x
* Mono.Android - 9.0

### PLATFORMS

- [X] Android
- [ ] iOS
- [ ] UWP

### ACTUAL BEHAVIOR

With latest version of Xamarin.GooglePlayServices (released yesterday), the map does not show up, it is replaced by a label "Xamarin.Forms.GoogleMaps" with a purple background.

I tested several scenario and it appears it works perfect with Xamarin.GooglePlayServices 60.x. Mono.Android 9.0 does not seem to be the guilty here.

### EXPECTED BEHAVIOR

Map showing up :)

**Comment by amay077** [comment]
Please check listed below:

### 1. Xamarin.FormsGoogleMaps.IsInitialized returns true? in your MainActivity.cs

![image](https://user-images.githubusercontent.com/401369/61638795-16150a80-acd5-11e9-929b-36cab65fa891.png)

### 2. Dump your Application Output

Is there any log for [Google Maps Android API] ?

```
[Google Maps Android API] Authorization failure.  Please see https://developers.google.com/maps/documentation/android-api/start for how to correctly set up the map.
[Google Maps Android API] In the Google Developer Console (https://console.developers.google.com)
[Google Maps Android API] Ensure that the "Google Maps Android API v2" is enabled.
[Google Maps Android API] Ensure that the following Android Key exists:
[Google Maps Android API] 	API Key: AIzaxxxxxxxx
[Google Maps Android API] 	Android Application (<cert_fingerprint>;<package_name>): D4:C0:xxxxxx;net.amay077.xfgooglemapsample
```

**Comment by nhdanh** [comment]
> Please check listed below:
> 
> ### 1. Xamarin.FormsGoogleMaps.IsInitialized returns true? in your MainActivity.cs
> ![image](https://user-images.githubusercontent.com/401369/61638795-16150a80-acd5-11e9-929b-36cab65fa891.png)
> 
> ### 2. Dump your Application Output
> Is there any log for [Google Maps Android API] ?
> 
> ```
> [Google Maps Android API] Authorization failure.  Please see https://developers.google.com/maps/documentation/android-api/start for how to correctly set up the map.
> [Google Maps Android API] In the Google Developer Console (https://console.developers.google.com)
> [Google Maps Android API] Ensure that the "Google Maps Android API v2" is enabled.
> [Google Maps Android API] Ensure that the following Android Key exists:
> [Google Maps Android API] 	API Key: AIzaxxxxxxxx
> [Google Maps Android API] 	Android Application (<cert_fingerprint>;<package_name>): D4:C0:xxxxxx;net.amay077.xfgooglemapsample
> ```

I have same problem when update google playservice. I debug follow check listed.

1. Xamarin.FormsGoogleMaps.IsInitialized returns false in MainActivity. 
2. No  any log for google map api android. 

I downgrade googleplay services. Map working. Can fix this problem with new version googleplayservices ?

**Comment by GuiF007** [comment]
Sorry I can't test much more right now, but if I remember well, I did not see any suspicious log about Google Map API. The simple patch is to revert to v. 60, without touching anything else...

**Comment by maxyhi** [comment]
I am having the same issue on the Huawei Mate 20 Pro, Xamarin.FormsGoogleMaps.IsInitialized returns true in MainActivity

**Comment by amay077** [comment]
I create small apps for latest libs and running on AndroidP emulator.

![image](https://user-images.githubusercontent.com/401369/62724851-e002b500-ba4e-11e9-8a81-9b5bc651c4ef.png)

Solution is here.

https://www.dropbox.com/s/zoet8ltmkkaxa8y/XFGoogleMapSample2.zip?dl=0

**You should change the API KEY in ``MyApp.cs``**

Could you try this apps and tell me what different with your apps?

See Also: https://github.com/google/volley/issues/200#issuecomment-403747481

**Comment by BillyMartin1964** [comment]
What is this MyApp.cs ?  I don't have one. If I made one, where would I call it from?

**Comment by amay077** [comment]
Android side's ``MyApp.cs`` is Application class for your Android apps.
This is called from Android OS when app launching.

* https://github.com/codepath/android_guides/wiki/Understanding-the-Android-Application-Class

**Comment by BillyMartin1964** [comment]
Has anyone got this working yet?

**Comment by BillyMartin1964** [comment]
I had to drop back to Xamarin.GooglePlayServices 60.x.  to make it work, so there I'll stay until you figure it out.

**Comment by chadgruka** [comment]
I experienced this issue, but found it was due to possible Multi-Dex setup and/or missing NuGet packages.

**Needed the 4 packages listed as installed**
![image](https://user-images.githubusercontent.com/10351344/64907886-88e3a600-d6c6-11e9-800d-8505d03efed6.png)

**Multi-Dex setup that worked**
![image](https://user-images.githubusercontent.com/10351344/64907889-9862ef00-d6c6-11e9-810f-5cbc1eb179ee.png)

Hope this helps people...

**Comment by foxanna** [comment]
I also had Google Maps gone on Android after updating `Xamarin.GooglePlayServices.*` packages from `v.60.1142.1` to `v.71.16xx.x`.   
  
Debug logs would show   
```
Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.maps.model.LatLng>: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/internal/zzbck;  
...  
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.android.gms.internal.zzbck" on path: DexPathList[[zip file ...]]
```
  
Downgrading `Xamarin.GooglePlayServices.*` packages back to `v.60.1142.1` fixed the issue, however, that solution was not acceptable.  
  
Thanks to [the previous comment](https://github.com/amay077/Xamarin.Forms.GoogleMaps/issues/639#issuecomment-531474533) by @chadgruka I was able to resolve this issue by adding `Xamarin.GooglePlayServices.Maps` and `Xamarin.GooglePlayServices.Location` of `v.71.16xx.x` to Android project, alongside with existing 
`Xamarin.GooglePlayServices.Base`, `Xamarin.GooglePlayServices.Basement`, `Xamarin.GooglePlayServices.Tasks`, `Xamarin.GooglePlayServices.Analytics` of `v.71.16xx.x`.

**Comment by mtsrdr** [comment]
The library dependencies requires the installation of Xamarin.GooglePlayServices.Maps library v.60.1142.1, but when you have others Xamarin.GooglePlayServices dependencies with a newer version like >= v.71.16xx.x, then the Maps is not loaded in the android project.
The same thing happened to me with the Xamarin.Forms.Maps library.

The solution, like @chadgruka said, is to install the latest version of Xamarin.GooglePlayServices.Maps.

**Comment by amay077** [comment]
I can't reproduce.

I updated Xamarin.GooglePlayServices.* to ver 71.1610.0, but apps works fine.

![image](https://user-images.githubusercontent.com/401369/66131803-0730c980-e62f-11e9-8f35-59f0bff2d1bc.png)

![image](https://user-images.githubusercontent.com/401369/66132050-6f7fab00-e62f-11e9-8eaf-4287bc607604.png)


My sample is here .

[XFGoogleMapSample.zip](https://github.com/amay077/Xamarin.Forms.GoogleMaps/files/3686310/XFGoogleMapSample.zip)

I can't investigate and fix it.

Could anyone give me minimum apps for reproduce?

**Comment by mtsrdr** [comment]
To help you I was able to reproduce the issue in the following way:

First I created a project and installed Xamarin.Forms.GoogleMaps on it. It was working fine, but I decided to install Xamarin.GooglePlayServices.Auth to create a Google Authentication Login, but the installation process failed asking me to install Xamarin.GooglePlayServices.Basement and Xamarin.GooglePlayServices.Base directly in the project.

So I installed the packages in the following order:
  Xamarin.GooglePlayServices.Basement
  Xamarin.GooglePlayServices.Base

If I tried to run the project again, the GoogleMaps will not be loaded in Android, unless I directly install Xamarin.GooglePlayServices.Maps latest version in my project.

I think the problem is that your library installs  Xamarin.GooglePlayServices.Basement v.60.1142.1 and Xamarin.GooglePlayServices.Base v.60.1142.1, but if I update theses packages, I have to update Xamarin.GooglePlayServices.Maps as well. And this is not done automatically. The only way to do is installing it directly.

I attached my sample project in order to help with the exception;

[TryMaps.zip](https://github.com/amay077/Xamarin.Forms.GoogleMaps/files/3687105/TryMaps.zip)
